testutil/docker.go

```diff
@@ -271,3 +271,24 @@ func DockerInspect(containerID string) (types.ContainerJSON, error) {
 	x.Check(err)
 	return cli.ContainerInspect(context.Background(), containerID)
 }
+
+// checkHealthContainer check health of container and determine wheather container is ready to accept request or not
+func CheckHealthContainer(socketAddrHttp string) error {
+	var err error
```

Comment:
do we need to declare these variables here?

Reply:
yes @mangalaman93 ,Cause we are using them ouside the loop.